recipes/oracledb/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "oracledb" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oracledb-{{ version }}.tar.gz
+  sha256: 9f938134e08a6d08f3886af90a0a9cee197dbe1b38ff7f75f1fc3262d27ba708
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython
+    - pip
+  run:
+    - python
+    - cryptography >=3.4
+
+test:
+  imports:
+    - oracledb
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://oracle.github.io/python-oracledb
+  summary: Python interface to Oracle Database
+  dev_url: https://github.com/oracle/python-oracledb
+  doc_url: https://python-oracledb.readthedocs.io/en/latest/
+  license: Apache-2.0
+  license_family: Apache
+  license_file:
+    - LICENSE.txt
+    - NOTICE.txt
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - nicoddemus
```

Comment:
@nicoddemus, it seems like you are the only active maintainer of https://github.com/conda-forge/cx_oracle-feedstock.  It seems that `oracledb` is just a rename (and new versioning) of `cx_oracle`.  Would you like to be a maintainer of the new package?

Reply:
Thanks xylar, but I will pass on this one: I already maintain a ton of packages and we stopped using cx-oracle. 😁

Reply:
Okay, no problem.